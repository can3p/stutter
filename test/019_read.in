(list
 (read "(car '(100 200 300))")
 (read 100)
 (read "(car '(10000 200 300)) (+ 1 1)"))
