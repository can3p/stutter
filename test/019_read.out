((car (quote (100 200 300))) "read expects a string argument" "read argument must contain exactly one form")
